runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/process.json

```diff
@@ -460,6 +460,7 @@
     "Process.Details.Modeler.ActionPanel.Form.ActionSystem.Tooltip.Title": "Runs on bots with {{ system }} system only",
     "Process.Details.Modeler.ActionPanel.Form.Connection.Expression": "Expression",
     "Process.Details.Modeler.ActionPanel.Form.Connection.Expression.Expression": "Expression",
+    "Process.Details.Modeler.ActionPanel.Form.Connection.Empty.Expression": "Expression can't be empty. Fill the field or remove sequence flow.",
```

Comment:
imo its better to have just `This field is required` or the same as in different inputs to be displayed under the element and add info icon next to it saying `Non-default flows must contain logical condition`
maybe even lets have a single information element in this form